src/toil/test/lib/dockerTest.py

```diff
@@ -123,72 +130,52 @@ def testDockerPipeChainErrorDetection(self, caching=True):
         if non-last commands in the chain fail.
         """
         options = Job.Runner.getDefaultOptions(os.path.join(self.tempDir, 'jobstore'))
-        options.logLevel = 'INFO'
+        options.logLevel = 'OFF'
         options.workDir = self.tempDir
         options.clean = 'always'
         if not caching:
             options.disableCaching = True
         A = Job.wrapJobFn(_testDockerPipeChainErrorFn)
         rv = Job.Runner.startToil(A, options)
         assert rv == True
-
-    def testDockerPermissions(self, caching=True):
```

Comment:
Why don't we need these two tests anymore?

Reply:
File permissions are already tested above (though perhaps it is a better idea to separate this from the orphan killing tests above). I'll re-implement this when I attempt to rewrite the main dockerClean test as multiple subtests.